02-express-tutorial/public/index.js

```diff
@@ -0,0 +1,11 @@
+document.getElementById("get-products").addEventListener("click", async () => {
```

Comment:
Pretty good!
I would just add an empty line after line 4 and that is it!

Reply:
Hi Sir ,
I have an autoformer on my VSCODE(settings.json) that removes empty spaces but i tried it directly on Github and it good now .
Thanks Sir !